index.rst

```diff
@@ -1,59 +1,367 @@
-..
-  Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Use the following syntax for sections:
+.. sectnum::
 
-  Sections
-  ========
+.. TODO: Delete the note below before merging new content to the master branch.
 
-  and
+.. note::
 
-  Subsections
-  -----------
+   **This technote represents a policy proposal that has not yet been adopted.**
 
-  and
+While the Gen3 Butler provides some intrinsic structure to its data repositories, considerably more is left to convention (often encoded in higher-level packages, like obs_base).  This document will be - at least at first - a proposal for how to organize data repositories in detail, focusing on collection naming conventions, filesystem locations, and developer workflows.  The immediate focus will be the environment at NCSA, but it is hoped that much of this will hold for the IDF and USDF as well.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+After consultation with other stakeholders and ultimately RFC, at least some of the content here should probably be moved to the DM Developer Guide.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+Collections
+===========
 
-     Caption text.
+The Gen3 Butler organizes datasets primarily via *collections*, which are groups of datasets defined strictly in the database (while some collections may relate directly to filesystem or other storage locations, this should usually be considered an implementation detail).
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+There are several types of collections, which are quite different in how they how they are written, but almost identical to readers.
+A full description of the types of collections can be found in the `daf_butler documentation`_, but a brief summary here should be sufficient for the proposals in this document:
 
-   Feel free to delete this instructional comment.
+- A ``RUN`` collection (or just "a run") is the only type of collection that is intrinsic to a dataset - a dataset is added to ``RUN`` collection when it is first added to a repository, and remains in that ``RUN`` collection until/unless it is deleted entirely.
```

Comment:
I agree that `RUN` is not a great name, though I think it's not a terrible one: it is the collection type that _always_ directly stores processing outputs, and in a long-lived repo those collections will greatly outnumber the "special" `RUN` collections that that are discussed more here (and that's where the name came from).  On the other hand, those special `RUN` collections will remain quite important, so maybe quantity isn't a great metric, but back on the original hand, users won't really interact with those special `RUN` collections directly, because we're setting up `CHAINED` collection pointers to them instead.

Overall, if I had a clearly better name in hand, I'd be tempted to see whether we could switch to it it without breaking anything (or at least without breaking repos; a lot of code would definitely have to change), but I'm not certain it's possible at this point.